src/pretix/control/templates/pretixcontrol/admin.html

```diff
@@ -0,0 +1,18 @@
+{% extends "pretixcontrol/base.html" %}
+{% load i18n %}
+{% load bootstrap3 %}
+
+{% block title %}{% trans "Admin settings" %}{% endblock %}
```

Comment:
Should we maybe call it "Global settings", as we don't really have a clear usage of the term "admin" yet (might be used for an *event admin* after the permissions refactor, I don't know yet)? I'm not sure about this.